jedi_vim.py

```diff
@@ -205,6 +205,17 @@ def completions():
         vim.command('return ' + strout)
 
 
+@contextmanager
+def tempfile(content):
+    # Using this instead of the tempfile module because Windows won't read
+    # from a file not yet written to disk
```

Comment:
That's what we were doing before which didn't work on Windows, unless I'm misunderstanding.